### PR TITLE
[bug 1082900] Filter KB dashboard by category

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

**Comment by mythmon** [comment]
Travis had an unrelated failure. I restarted the job.

**Comment by mythmon** [comment]
This time the Travis failure is an actual test failure. :boom:

**Comment by rehandalal** [comment]
gah! i am silly! fixed ^

**Comment by safwanrahman** [comment]
I think this button should also be in Complete overview Page. So a person can see all the document of a category in Complete overview Dashboard.
`/en-US/contributors/kb-overview`

**Comment by safwanrahman** [comment]
I think the UI looks better if its like this. 
firefox_screenshot_2015-07-14t20-45-37 701z

File: kitsune/dashboards/views.py [hunk 1]
```diff
@@ -210,3 +215,17 @@ def _get_product(request):
...
     return None
+
+
+def _get_category(request):
+    category = request.GET.get('category')
+
+    if category:
+        for c in CATEGORIES:
+            print c[0]
+            print category
```
Comment (mythmon) [inline kitsune/dashboards/views.py]: Oops!

**Comment by mythmon** [comment]
I think Safwan is right about the visual part of this, it would be better if it was horizontally aligned with the subscribe button. Right now it looks kind of haphazard.

Functionality seems to work great. You've got a couple debug print statements, but besides that, the code looks good.

I think we don't need to add it to the other page right now. It makes sense, but that would be a different bug, and it makes sense to make it a different PR to keep things small.

**Comment by rehandalal** [comment]
fixed ^

**Comment by mythmon** [comment]
r+

**Pull request merged**